src/configs/index.js

```diff
@@ -143,6 +143,62 @@ export const errorMessages = [
   }
 ];
 
+export const SORT_BY_SELECT_OPTIONS = [
```

Comment:
Please, move translations out of the `configs` to the `translations` folder

Reply:
fixed